### PR TITLE
Video Library

### PR DESCRIPTION
**Pull request by percius47**
Features:
-video Listing
-categories
-like, watch later, history
-auth: login, signup
-playlist management
[Live Site](https://wideo.netlify.app/)

### CODE REVIEW

**Review by KoushikKintali — COMMENTED** [review]
Please drop a small PR's
It is very difficult to review as there are 94 file changes

File: wideo/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
```
Comment (Varu98) [inline wideo/public/index.html:26]: maybe these comments can be removed later on.

File: wideo/src/pages/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  <h2 class="login-heading">Signup</h2>
+  <div class="login-inputs">
+<form  onSubmit={(e) =>
+				signUpHandler(
+					e,
+					formVal.email,
+					formVal.password,
+					formVal.firstName,
+					formVal.lastName
+				)
```
Comment (Varu98) [inline wideo/src/pages/Signup.jsx:51]: maybe formVal can be improved as a naming convention

File: wideo/src/pages/SingleVideo.jsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+import { addToLikes } from '../services/likes/addLikesService';
+import { deleteLikes } from '../services/likes/deleteLikes';
+import { removeWatchLater } from '../services/watchlist/removeWatchLater';
+import { addToWatchLater } from '../services/watchlist/watchLater';
+import { checkPlaylist } from '../util/checkPlaylist';
+import ThumbUpOutlinedIcon from '@mui/icons-material/ThumbUpOutlined';
+import ThumbUpIcon from '@mui/icons-material/ThumbUp';
+import LibraryAddRoundedIcon from '@mui/icons-material/LibraryAddRounded';
+import BookmarkRoundedIcon from '@mui/icons-material/BookmarkRounded';
+import BookmarkBorderRoundedIcon from '@mui/icons-material/BookmarkBorderRounded';
```
Comment (Varu98) [inline wideo/src/pages/SingleVideo.jsx:19]: maybe we can use barrel imports here.

**Review by Varu98 — APPROVED** [review]

**Pull request merged**